src/js/view/view.js

```diff
@@ -847,12 +849,14 @@ define([
             clearTimeout(_controlsTimeout);
             _controlbar.hideComponents();
             utils.addClass(_playerElement, 'jw-flag-user-inactive');
+            _captionsRenderer.renderCues();
         }
 
         function _userActivity() {
             if(!_showing){
                 utils.removeClass(_playerElement, 'jw-flag-user-inactive');
                 _controlbar.checkCompactMode(_videoLayer.clientWidth);
+                _captionsRenderer.renderCues();
```

Comment:
Can't we just depend on CSS for this? What is being recalculated when the controlbar is shown?